solr/conf/slug.js

```diff
@@ -0,0 +1,77 @@
+function processAdd(cmd) {
+  var doc = cmd.solrDoc;
+  var lang = doc.getFieldValue("detectlang_s");
+  var title = doc.getFieldValue("title_txtm");
+  var maxWords = 5;
+  var maxCharacters = 35;
+
+  // Remove punctuation, and split into words
+  var words = title
+              .replace(/[.,\/#!$%\^&\*'";:{},<=>\-_@`~\[\]()]/g,'')
+              .replace(/  +/g, ' ')
+              .toLowerCase()
+              .split(' ')
+
+  // Filter out stopwords
+  if (lang in stopwords) {
+    filtered_words = words
+                    .filter(function(el) { return stopwords[lang].indexOf(el) == -1 })
```

Comment:
I'm not sure what `el` means. maybe element? Maybe we could change this to `word`?